src/sentry/static/sentry/app/redux/actions/workBatch.js

```diff
@@ -1,49 +1,11 @@
-import axios from 'axios';
+import {Client} from 'app/api';
```

Comment:
There are going to be a bunch of action reducers in future. What about organizing them, so that they are placed in the same folder as the main view calling them? This would be for a practical point of view. When updating them, I have otherwise have to look in 3 separate places which each are to be filled up with a lot of files.

Reply:
I wouldn't be against doing that, organizing both the actions and the views together based on main use case. This was setup by Mariya which I think prefers this, and I'm not against that either. So perhaps we can take it up again later.